springfield/cms/templates/pattern-library/base-styles/an_introduction/FLARE_COMPONENTS.md

```diff
@@ -0,0 +1,145 @@
+# Flare Components
+
+File structure and patterns for creating Flare components in Springfield's Wagtail CMS.
+
+## Component Anatomy
+
+Every Flare component has 4 parts:
+
+1. [Base Template](#1-base-template) - Component markup and structure
+2. [Component CSS](#2-component-css) - Styles and variations
+3. [Wagtail Block](#3-wagtail-block) - CMS data bridge
```

Comment:
The corresponding IDs for these links are not being inserted so the links don't work.